theme/templates/pages/group.html

```diff
@@ -569,10 +580,57 @@ <h4 class="panel-title">
 
                             {% include "includes/legend.html" %}
 
-                        </div>
-                     </div>
-                 </div>
+                         {% else %}
```

Comment:
If not a member of the Group provide option to join or display join request status